src/hotspot/share/sanitizers/address.hpp

```diff
@@ -29,6 +29,25 @@
 #include <sanitizer/asan_interface.h>
 #endif
 
+// ATTRIBUTE_NO_SANITIZE_ADDRESS
```

Comment:
Can we find a better name for this? Its lengthy but still not very clear. E.g. as a prefix to a function it reads like it does something to the - should there be one - return pointer of the function.

proposal: ASAN_DISABLE

Reply:
I was mostly going for being consistent with other uses in globalDefinitions. Most things are ATTRIBUTE_X where X is the actual attribute name.

Maybe ATTRIBUTE_NO_ASAN if we want something shorter?

Reply:
Sounds good.

Reply:
Renamed.